M2/Macaulay2/d/expr.d

```diff
@@ -164,11 +164,15 @@ export newHashTable(Class:HashTable,parent:HashTable):HashTable := (
 	  -- we start with four empty buckets.  It is important for the 
 	  -- enlarge/shrink code in hashtable.dd that the number of buckets
 	  -- (here four) is a power of two
-	  Class,parent,0,nextHash(),
+	  Class,parent,0,0,
```

Comment:
Are there any hash tables that might sneak by with a hash of 0 after this change?  Or is it always set to something else elsewhere?

Reply:
this needs to be checked carefully, but I believe the answer is yes. `newHashTable(...)` is called by `makenew` which sets the hash, and by a bunch of other functions in say `actors4.d` and `evaluate.d` which call `sethash` afterwards.